README.md

```diff
@@ -440,12 +471,14 @@ You can read more about special partners and these integrations in our
 [event-tracking]: https://docs.adjust.com/en/event-tracking
 [special-partners]: https://docs.adjust.com/en/special-partners
 [currency-conversion]: https://docs.adjust.com/en/event-tracking/#tracking-purchases-in-different-currencies
+[universal-links-guide]: https://docs.adjust.com/en/universal-links/
+[universal-links]: https://developer.apple.com/library/ios/documentation/General/Conceptual/AppSearch/UniversalLinks.html
 
 ## License
 
 The adjust SDK is licensed under the MIT License.
 
-Copyright (c) 2012-2015 adjust GmbH,
+Copyright (c) 2012-2016 adjust GmbH,
 http://www.adjust.com
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
```

Comment:
FREEDOM'FREEDOM Oh yes! FREEDOM'FREEDOM Oh yes, yes, yes!
FREEDOM'FREEDOM wOw YES!!!
Thankful to be Grateful,,,
We are1FOREVER 
We have jUan(wOn)2geder 
WISDOM'PASSION'LEGACY